README.md

```diff
@@ -0,0 +1,122 @@
+# Action Server
+
+The action server is an actionlib server for managing the execution of high level tasks and their semantic chaining.
+It takes actionlib goals describing series of high level tasks in json form, checks the consistency of the semantics, and executes them.
+This readme explains the architecture of the action server, the procedure of handling a goal and the lifecycle of an action.
+
+## Architecture
+
+![Action Server architecture](doc/action_server_architecture.jpg)
+
+The action server uses an actionlib interface for communication with its clients.
+A client implementation is provided as abstraction from the raw actionlib client.
+Current examples of clients to the action server are:
+ - The GPSR challenge
+ - The Natural Language Console
+Consult one of these implementations, or the Client docstring for example usage of the Client class.
```

Comment:
add extra linebreak (sentence shows up in the bullet list)

Reply:
Done :)